career_development/ladder.md

```diff
@@ -13,102 +13,112 @@
 ## Dev Values : Simple, Test-Driven, Iterative, Communicative, Respectful, Skilled 
 
 ### Level 1
-
-#### simple 
--[ ] is working on naming variables and learning to extract methods
+#### Simple 
+-[ ] Is working on naming variables and learning to extract methods.
 	
-#### test-driven
--[ ] runs all tests before commiting, can understand where expectations are not met and fix some with the help of more senior developers
+#### Test-driven
+-[ ] Runs all tests before commiting.
+-[ ] Can understand where expectations are not met and fix some with the help of more senior developers.
 	
-#### iterative
--[ ] understands red-green-refactor
+#### Iterative
+-[ ] Understands red-green-refactor.
 	
-#### communication
--[ ] updates asana, comes prepared for standup, uses time tracking tools and vacation notifications
+#### Communication
+-[ ] Updates Asana.
+-[ ] Comes prepared for meetings.
+-[ ] Uses time tracking tools and vacation notifications.
 	
-#### respectful teammate
--[ ] on time for meetings, is enthusiastic
+#### Respectful teammate
+-[ ] On time for meetings.
+-[ ] Is enthusiastic.
 	
-#### hard skills
--[ ] can use git, linters, makes clear prs
+#### Hard skills
+-[ ] Can use git, linters, makes clear prs.
```

Comment:
In my opinion a "clear pr" is a PR that:
- Is well described
- Doesn't have side effects
- Has clearly described commits

I feel like that's too much to ask of a level one person... maybe "makes prs" is a better place to start? Or am I expecting way too less?

Reply:
What if we move it to level 2? 
Anyone can open a PR, so I think it's too soft of a requirement.

Reply:
`Is well described` and `contains asana link` might be the two we care about at level 1